ggipcd/unit/ggl.gg-ipc.socket.socket.in

```diff
@@ -0,0 +1,19 @@
+[Unit]
```

Comment:
`socket.socket` in the file name, is that intentional?

Reply:
I don't think it is required. It is a typo. I will fix it.

Reply:
I reverted this change. As the file name is socketName.socket.
Here in this case the socketName is ggipc.socket. That is why the file name is ggipc.socket.socket